src/SFA.DAS.EmployerIncentives.Web/Services/Applications/ApprenticeshipIncentiveService.cs

```diff
@@ -19,24 +21,45 @@ public ApprenticeshipIncentiveService(HttpClient client, IHashingService hashing
             _hashingService = hashingService;
         }
 
-        public async Task<GetApplicationsModel> GetList(string accountId, string accountLegalEntityId)
+        public async Task Cancel(string accountLegalEntityId, IEnumerable<ApprenticeshipIncentiveModel> apprenticeshipIncentives)
+        {
+            var decodedAccountLegalEntityId = _hashingService.DecodeValue(accountLegalEntityId);
+
+            var url = $"withdrawals";
```

Comment:
Does this endpoint withdraw the apprenticeships immediately? At refinement we discussed that this should be an asynchronous process/message based, so that withdrawals aren't processed while the payments month end is in progress.